src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -7,9 +7,11 @@ import useGetResponsesAndElementState from '../../utils/hooks/useGetResponsesAnd
 import { ElementsBox, NavigationBox } from './'
 import validatePage, { getCombinedStatus, PROGRESS_STATUS } from '../../utils/helpers/validatePage'
 import { SummarySectionCode } from '../../utils/constants'
+import getPageElements from '../../utils/helpers/getPageElements'
```

Comment:
Not required. I just thought it would be better to be done outside this presentation component. I can remove this change if you think would conflict too much with other work.

Reply:
No it's all good. :)